packages/create-app/templates/template-next-static-export/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES2020",
```

Comment:
This is to match the other templates based on vite which originally was needed when I had some of the generated code local to the example packages but might not matter anymore now that I reused an existing local package